platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/AttributionDialogManager.java

```diff
@@ -138,21 +143,61 @@ public void onClick(@NonNull DialogInterface dialog, int which) {
     builder.show();
   }
 
-  private void showMapFeedbackWebPage(int which) {
+  private void showMapAttributionWebPage(int which) {
     Attribution[] attributions = attributionSet.toArray(new Attribution[attributionSet.size()]);
     String url = attributions[which].getUrl();
-    if (url.contains(MAP_FEEDBACK_URL)) {
-      url = buildMapFeedbackMapUrl(mapboxMap.getCameraPosition());
+    Logger.d(TAG, "attribution url: " + url);
+    if (url.contains(MAP_FEEDBACK_URL_OLD) || url.contains(MAP_FEEDBACK_URL)) {
+      url = buildMapFeedbackMapUrl(Mapbox.getAccessToken());
     }
+    Logger.d(TAG, "final feedback url: " + url);
     showWebPage(url);
   }
 
   @NonNull
-  private String buildMapFeedbackMapUrl(@Nullable CameraPosition cameraPosition) {
-    // appends current location to the map feedback url if available
-    return cameraPosition != null ? String.format(Locale.getDefault(),
-      MAP_FEEDBACK_LOCATION_FORMAT, cameraPosition.target.getLongitude(), cameraPosition.target.getLatitude(),
-      (int) cameraPosition.zoom) : MAP_FEEDBACK_URL;
+  String buildMapFeedbackMapUrl(@Nullable String accessToken) {
+    // TODO Add Android Maps SDK version to the query parameter, currently the version API is not available.
```

Comment:
since we have a ticket tracking this, I would suggest adding a reference to the ticket as `#1234`. 
This can be used to look up the ticket at a later time.

Reply:
Added the ticket number and link to the TODO.